Tests/Tests/EventMatcherSpec.swift

```diff
@@ -212,6 +227,16 @@ class EventMatcherSpec: QuickSpec {
                 expect(events).to(contain(AppStartEvent()))
             }
 
+            it("will stop matching events when triggers inside campaign is nil") {
```

Comment:
I'm not sure 'stop' is a good verb here. How about "will not match any event with a campaign that has no triggers"

Reply:
Good point, I will change the description.